scripts/processsourcefiles.sh

```diff
@@ -97,40 +105,14 @@ source scripts/convert-repo-ulrs.sh
 #########################################################
 # Process content in .md files (MAKE RELATIVE LINKS WORK)
 # We want users to be able view and use the source files in GitHub as well as on the site.
-# Therefore, the following changes need to be made to all docs files prior to Hugo site build.
-# Convert GitHub enabled source, into HUGO supported content:
-#  - For all Markdown files under the /content/ directory:
-#    - Skip all:
-#      - Markdown link with fully qualified HTTP(s) URL is 'external'
-#      - GitHub file (.git* files)
-#      - non-docs directories
-#    - Ignore Hugo site related files (avoid "readfile" shortcodes):
-#      - _index.md files (Hugo 'section' files)
-#      - API shell files (serving-api.md, eventing-contrib-api.md, eventing-api.md)
-#    - For all remaining Markdown files:
-#      - Remove all '.md' file extensions from within Markdown links "[]()"
-#      - For SEO convert README to index:
-#       - Replace all in-page URLS from "README.md" to "index.html"
-#       - Rename all files from "README.md" to "index.md"
-#      - Adjust relative links by adding additional depth:
-#       - Exclude all README.md & _index.md files
-#       - Convert './' to '../'
-#       - Convert '../' to '../../'
-echo 'Converting all links in GitHub source files to Hugo supported relative links...'
-# Convert relative links to support Hugo
-find . -type f -path '*/content/*.md' ! -name '*_index.md' ! -name '*README.md' \
-    ! -name '*serving-api.md' ! -name '*eventing-contrib-api.md' ! -name '*eventing-api.md' \
-    ! -name '*build-api.md' ! -name '*.git*' ! -path '*/.github/*' ! -path '*/hack/*' \
-    ! -path '*/node_modules/*' ! -path '*/test/*' ! -path '*/themes/*' ! -path '*/vendor/*' \
```

Comment:
Im very close to Go illiterate but I didnt see these files obviously ignored up in the Hugo layout? 

For anything here that is explicitly called out, its because Hugo will build that file into the site and add empty/no title Nav entries into our TOC. They rendered as blank spaces and are clickable but since they lacked front matter (to either add the missing meta or explicitly exclude it from the build) they get added to the site in unwanted and conflicting ways.

Reply:
I figured out what these actually mapped to and made sure that they rendered correctly as best as I could tell.